Lombiq.Privacy.Samples/Navigation/PrivacySamplesNavigationProvider.cs

```diff
@@ -0,0 +1,21 @@
+using Lombiq.HelpfulLibraries.OrchardCore.Navigation;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Localization;
+using OrchardCore.Navigation;
+
+namespace Lombiq.Privacy.Samples.Navigation;
+
+public class PrivacySamplesNavigationProvider : MainMenuNavigationProviderBase
+{
+    public PrivacySamplesNavigationProvider(
+        IHttpContextAccessor hca,
+        IStringLocalizer<PrivacySamplesNavigationProvider> stringLocalizer)
+        : base(hca, stringLocalizer)
+    {
+    }
+
+    protected override void Build(NavigationBuilder builder) =>
+        builder
+            .Add(T["Privacy Samples"], builder => builder
+                .Add(T["Competitor registration"], itemBuilder => itemBuilder.Url("/competitor-registration")));
```

Comment:
I meant put both links into here, so all the relevant pages are available from the same main menu hub. You don't need the separate admin menu.

Reply:
I don't understand. Why we want to bring an admin function out to a non admin menu?

Reply:
This is a demo site, the purpose of the menu is to guide a user to the points of interest for that module so they don't have to hunt through the source code to find what they need to look for.